src/conv-core/converse.h

```diff
@@ -780,32 +780,18 @@ CpvExtern(int, _curRestartPhase);      /* number of restarts */
 /** This header goes before each chunk of memory allocated with CmiAlloc. 
     See the comment in convcore.C for details on the fields.
 */
-struct CmiChunkHeader {
+struct
+#if !(CMK_USE_IBVERBS | CMK_USE_IBUD)
+  alignas(ALIGN_BYTES) // If in verbs mode, align in infiCmiChunkHeader instead
+#endif
+CmiChunkHeader {
   int size;
```

Comment:
Isn't leaving the CmiChunkHeader size field an int going to result in some buggy issues when you try to allocate large messages?

Reply:
Yes, it will, but this PR isn't meant to fix that. Rather, these are some underlying issues that are getting in the way of a larger change to use the right types to fix large messages.

I'm working on another PR to support large messages properly, this just came up as a problem while I was doing that.